src/PactNet.AspNetCore/Messaging/MessageMiddleware.cs

```diff
@@ -99,7 +99,7 @@ private static void AddMetadataToResponse(HttpContext context, dynamic metadata)
             string metadataBase64 = Convert.ToBase64String(Encoding.UTF8.GetBytes(stringifyMetadata));
 
             context.Response.ContentType = "application/json";
-            context.Response.Headers.Add("PACT_MESSAGE_METADATA", metadataBase64);
+            context.Response.Headers.Add("pact-message-metadata", metadataBase64);
         }
```

Comment:
Did the FFI upgrade necessitate this? If so, can you link to the relevant FFI section?

Reply:
https://github.com/pact-foundation/pact-reference/commit/b3a6f193

Reply:
See also https://github.com/pact-foundation/pact-reference/tree/master/rust/pact_verifier_cli#verifying-metadata. Those docs really ought to be moved. Thanks for picking it up @Hoffs !

Reply:
Updated in #350